docs/src/pages/company/careers/designer.md

```diff
@@ -88,10 +88,10 @@ If your application stands out, we'll contact you for a get-to-know conversation
 
 If that goes well, you'll be invited to up to four additional conversations, those being:
 
-- A walkthrough on one or two selected projects of your career that you believe demonstrates [comparable experience.](https://articles.uie.com/ux-hiring-lets-talk-about-comparable-experience/)
+- A walkthrough on one or two selected projects of your career that you believe demonstrate [comparable experience.](https://articles.uie.com/ux-hiring-lets-talk-about-comparable-experience/)
 - A chat with one of the product managers or engineers that you will work closely with.
 - A chat with one of MUI's founders.
-- A possible follow-up chat if we missed addressing anything relevant in the previous conversations.
+- A possible follow-up chats if we missed addressing anything relevant in the previous conversations.
```

Comment:
```suggestion
- Possible follow-up chats if we missed addressing anything relevant in the previous conversations.
```

So the plural is right

Reply:
Ideally, there would be a single follow-up chat

```suggestion
- A possible follow-up chat if we missed addressing anything relevant in the previous conversations.
```

But then, Grammarly asks for the plural. @samuelsycamore which one is it, should we trust Grammarly or not?